features/display-contents.yml

```diff
@@ -0,0 +1,10 @@
+name: "display: contents"
+description: "The `display: contents` CSS declaration sets an element to display only its contents, not itself. The element does not generate a display box and may lose semantic or accessibility features."
+spec: https://drafts.csswg.org/css-display-3/#box-generation
+# caniuse: css-display-contents
+group: css
+status:
+  compute_from: css.properties.display.contents
```

Comment:
Continuing from https://github.com/web-platform-dx/web-features/pull/1966/files#r1803434536

Let's remove this:

```suggestion
```